Apps/W1/HybridGP/app/src/Migration/Support/CheckBooks/GPCheckbookMigrator.codeunit.al

```diff
@@ -11,6 +11,9 @@ codeunit 40025 "GP Checkbook Migrator"
         GPCompanyAdditionalSettings: Record "GP Company Additional Settings";
         MigrateInactiveCheckbooks: Boolean;
     begin
+        if not GPCompanyAdditionalSettings.GetBankModuleEnabled() then
```

Comment:
I don't think this is needed as you have the same check in the helper functions which calls this procedure.

Reply:
Yes, I missed this one.